sonosco/datasets/audio_dataset.py

```diff
@@ -69,16 +73,22 @@ def parse_audio(self, audio_path):
         if self.augment:
             sound = self.augment_audio(sound)
 
+        if raw:
```

Comment:
shouldn't this be before augmentation?

Reply:
By raw here I mean that the signal and not it's spectrogram is returned by this function, and the signal should still be augmented. Maybe the naming is a bit confusing.